lib/event-stream.js

```diff
@@ -269,4 +296,38 @@ EventStream.prototype._read = function() {
 	this.getLongPollInfo();
 };
 
+/**
+ * Implementation of stream-internal `_destroy` function (v8.0.0 and later).
+ * Called by stream consumers to effectively stop polling via the public
+ * `destroy()`.
+ * @returns {void}
+ * @private
+ */
+EventStream.prototype._destroy = function() {
+
+	if (!this.destroyed) {
+		/* istanbul ignore else */
+		if (typeof this.destroy === 'function') {
+			this.destroy();
```

Comment:
The [Node.js Stream docs](https://nodejs.org/api/stream.html#stream_readable_destroy_err_callback) say that `_destroy()` is called by `destroy()` — why call back into `destroy()` here?

Reply:
Because `destroy` was not added until v8 of Node.

Reply:
(so if you called `abort` you might as well have called `destroy` if it exists. this is moot if I just conditionally add the method)